frontend/lib/map/map_page.dart

```diff
@@ -39,6 +39,14 @@ abstract class MapPageController {
 class _MapPageState extends State<MapPage> implements MapPageController {
   MapController? _controller;
 
+  @override
+  void initState() {
+    super.initState();
+    WidgetsBinding.instance.addPostFrameCallback((timeStamp) async {
```

Comment:
❓ Are you sure that we need the `addPostFrameCallback` here
Afaics its not required that the ui is ready when the permission will be requested.
Did you face any issues without that?

Reply:
For me it actually does not show a permission request without this. Could you perhaps check and verify this?

Reply:
in theory it would be working but we have to wait for translation provider. So its okay
Another issue is now that the `followUserLocation` has to be retriggered.
At least on ios it will not be executed after granting permission

Reply:
i'm also getting an error
```js
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: A request for location permissions is already running, please wait for it to complete before doing another request.
#0      GeolocatorApple.requestPermission (package:geolocator_apple/src/geolocator_apple.dart:63:7)
<asynchronous suspension>
#1      checkAndRequestLocationPermission (package:ehrenamtskarte/location/determine_position.dart:129:29)
<asynchronous suspension>
#2      _MapPageState.initState.<anonymous closure> (package:ehrenamtskarte/map/map_page.dart:46:7)
<asynchronous suspension>

```

Reply:
It's also a bit weird, that we would ask for permission every time the user opens the app, even though, the user might not *want* to give us permission; so he has to dismiss this dialog every time.
Imo, it would then be better to just wait for the user to click on the `location_button`.

Reply:
thats a good point @michael-markl but apple requested this explicitly.
So if we say we only request permissions if the user clicked on the location button of the map it might not be convenient for them.

I updated them pr so this is outdated.